cmd/login/auth.go

```diff
@@ -24,9 +24,9 @@ type authInfo struct {
 	refreshToken string
 }
 
-// storeCredentials stores the installation's CA certificate, and
+// storeMCCredentials stores the installation's CA certificate, and
```

Comment:
I allowed myself to change the names of these commands to make them less confusing